src/components/card-recommender/card-recommender.tsx

```diff
@@ -186,7 +186,7 @@ function CardRecommenderInner(
   const { data, quantities, resolvedDeck, listState, getListCardProps } = props;
 
   const { t } = useTranslation();
-  const [theme] = useColorTheme();
+  const theme = getColorThemePreference();
```

Comment:
With the current implementation, this is fine, but this would now fail if we add a quick action / hotkey to change the theme - so using a hook is better here.